benchexec/resources.py

```diff
@@ -118,6 +118,24 @@ def _get_cpu_cores_per_run0(coreLimit, num_of_threads, allCpus, cores_of_package
     if coreLimit * num_of_threads > len(allCpus):
         sys.exit("Cannot run {0} benchmarks in parallel with {1} CPU cores each, only {2} CPU cores available. Please reduce the number of threads to {3}.".format(num_of_threads, coreLimit, len(allCpus), len(allCpus) // coreLimit))
 
+    if not use_hyperthreading:
+        logging.debug("Running in no-hyperthreading mode")
```

Comment:
I would suggest to move the logging to the end of the if statement, and then let it print something like `Running in no-hyperthreading mode, avoiding the use of CPU cores x, y, z.`

That might be useful for users.

Reply:
Refined the debug statement and moved it at the end of the if block